### PR TITLE
fix(husky): updated husky and now pre-commit hooks should work again

### PR DESCRIPTION
**Pull request by MarcMcIntosh**

### CODE REVIEW

**Comment by MarcMcIntosh** [comment]
Note after merging this and v4.2, run `staandard-changelog -f` to catch up on old commits :)

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (Miniplop) [inline .husky/pre-commit:1]: This file contain only bash comment, is it relevant to keep it ?

**Review by Miniplop — APPROVED** [review]
Great 👍

File: .husky/pre-commit (hunk 1 above)
Comment (MarcMcIntosh) [reply .husky/pre-commit:1]: Added lint-staged :)

**Pull request merged**